runners/null-runner/runner.go

```diff
@@ -0,0 +1,88 @@
+package runner
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/tinyci/ci-agents/clients/log"
+	"github.com/tinyci/ci-agents/clients/queue"
+	"github.com/tinyci/ci-agents/errors"
+	"github.com/tinyci/ci-runners/fw"
+	"github.com/tinyci/ci-runners/fw/config"
+)
+
+// Runner encapsulates an infinite lifecycle overlay-runner.
+type Runner struct {
+	Config    *config.Config
+	NextState bool
+}
+
+// Init is the bootstrap of the runner.
+func (r *Runner) Init(ctx *fw.Context) *errors.Error {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Why `UnixNano` and not just `Unix`? Not a problem or at all important, just curious.

Reply:
UnixNano(): https://godoc.org/time#Time.UnixNano

Unix has a granularity of one second, UnixNano is harder to repeat a seed for in a quick restart loop.